app/src/main/java/com/duckduckgo/app/settings/SettingsActivity.kt

```diff
@@ -68,8 +68,10 @@ class SettingsActivity : DuckDuckGoActivity() {
     override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
         when(requestCode) {
             REQUEST_CODE_ABOUT_DDG -> {
-                setResult(AboutDuckDuckGoActivity.RESULT_CODE_LOAD_ABOUT_DDG_WEB_PAGE)
-                finish()
+                if(resultCode == AboutDuckDuckGoActivity.RESULT_CODE_LOAD_ABOUT_DDG_WEB_PAGE ) {
```

Comment:
Not sure the when / if combination works well here. Can we simplify this into something along the lines of:
`if (requestCode == REQUEST_CODE_ABOUT_DDG && resultCode == AboutDuckDuckGoActivity.RESULT_CODE_LOAD_ABOUT_DDG_WEB_PAGE )`